dev/ci/ci-common.sh

```diff
@@ -135,6 +136,9 @@ git_download()
     if [ "$WITH_SUBMODULES" = 1 ]; then
         git submodule update --init --recursive
     fi
+    if [ "$CI" ]; then
```

Comment:
would
```suggestion
    if [ ! -z "$CI" ]; then
```
be more portable / robust to, e.g., `CI` being `=`?  Also maybe we should uniformize empty checking so we don't do `[ "$CI" = "" ]`, `[[ $ov_url ]]`, `[[ $CI ]]`, and `[ "$CI" ]`

Reply:
It's fine this way. We require bash so there's no portability concern.